product_by_supplier/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product by supplier info
+========================
+
+This module categorizes each product item by supplier.It allows for users to be able to view a compiled list of products supplied by the supplier.
```

Comment:
You can add that this info is the corresponding to the supplier list on each product.